js/block.js

```diff
@@ -887,6 +887,12 @@ ValueProto.createdCallback = function valueCreated(){
             if (this.hasAttribute('max')){
                 input.setAttribute('max', this.getAttribute('max'));
             }
+            if (this.hasAttribute('oninput')){
```

Comment:
This raises an interesting issue. Normally in this project we use event delegation (using `Event.on()`) and never set event handler attributes on the HTML. I don't immediately see a good way to do this in this case, though. Perhaps we need to set another attribute, like 'subtype` for use with validation handlers?